app/controllers/menus_controller.rb

```diff
@@ -1,10 +1,24 @@
 class MenusController < ApplicationController
-  skip_before_action :authenticate_employee!, only: [:home]
+  skip_before_action :authenticate_employee!, only: [:home, :search_email]
 
   def index
   end
 
-  def home
-    @companies = Company.all
+  def home   
   end
-end
\ No newline at end of file
+
+  def search_email
+    @email = params[:email]
+    if(@email != nil)
+      employees = Employee.where(email: params[:email])
+      @companies = []
+      employees.each do |employee|
+        company = Company.find_by(id: employee.company_id)
```

Comment:
`current_employee.company`

Reply:
nope all companies required plus current_employee not set yes